requirements.master.txt

```diff
@@ -1 +1,3 @@
 tx-manager>=0.2,<0.3
```

Comment:
@richmahn The tag could be anything, ir even just `@master` assuming the master branch is always the current release code.

And I don't think we need to do anything in setup.py. That is what pip would use to install tx-manager-lambda.